daemon/logger/plugin_unix.go

```diff
@@ -12,7 +12,10 @@ import (
 )
 
 func openPluginStream(a *pluginAdapter) (io.WriteCloser, error) {
-	f, err := fifo.OpenFifo(context.Background(), a.fifoPath, unix.O_WRONLY|unix.O_CREAT|unix.O_NONBLOCK, 0700)
+	// open with read/write so avoid broken pipes in case of a plugin failure.
```

Comment:
typo: "so" -> "to".
Also, open with write is obvious. the read is the non-obvious part and causing the broken pipe. So maybe reword to emphasize that?

Reply:
Will do.